package.json

```diff
@@ -10,7 +10,9 @@
   "scripts": {
     "start:dev": "node_modules/.bin/webpack-dev-server --config=webpack.config.js --host 0.0.0.0",
     "build": "NODE_ENV=production node_modules/.bin/webpack --config=webpack.config.js",
-    "deploy:now": "(cd dist && now)"
+    "deploy:now": "(cd dist && now)",
```

Comment:
You don't have to put it in a parenthesis. Also, what's the "now" command meant to be?

Reply:
Yeah not quite sure where that came from. but fixed it now